documentation/docs/steps/transportRequestUploadSOLMAN.md

```diff
@@ -32,9 +32,9 @@ only the truststore of the environment needs to be extended.
 
 The target of the upload is a Transport Request, which is determined by the identifiers (ID)
 of the Request and the associated Change Document.
-`transportRequestUploadSOLMAN` allows to set these IDs by parameter or to use Git commit messages.
-As an additional option, IDs can be passed in via the Common Pipeline Environment. For example through a
-step that generates the IDs or obtains them differently.
+`transportRequestUploadSOLMAN` allows to set these IDs by parameter.
+As an additional option, IDs can be passed in via the Common Pipeline Environment.
+For example through a step that generates the IDs or obtains them differently.
```

Comment:
For example, by performing a step that generates the IDs or obtains them differently. (vielleicht creates them differently?)